lib/output.js

```diff
@@ -148,11 +148,14 @@ function withMetadata (withMetadata) {
  * @param {Boolean} [options.overshootDeringing=false] - apply overshoot deringing, requires mozjpeg
  * @param {Boolean} [options.optimiseScans=false] - optimise progressive scans, forces progressive, requires mozjpeg
  * @param {Boolean} [options.optimizeScans=false] - alternative spelling of optimiseScans
+ * @param {Boolean} [options.optimiseCoding=true] - optimise Huffman coding tables
+ * @param {Boolean} [options.optimizeCoding=true] - alternative spelling of optimiseCoding
  * @param {Boolean} [options.force=true] - force JPEG output, otherwise attempt to use input format
  * @returns {Sharp}
  * @throws {Error} Invalid options
  */
 function jpeg (options) {
+  this._setBooleanOption('jpegOptimiseCoding', true);
```

Comment:
Please can this move to the default options in [constructor.js](https://github.com/lovell/sharp/blob/master/lib/constructor.js#L188)